server/api/main.js

```diff
@@ -0,0 +1,30 @@
+import { readFile } from 'fs'
+import { URL } from 'node:url'
+
+let responseCache = null
+
+export default async function handleMain(req, res) {
+  let filePath = new URL(`../../client/dist/index.html`, import.meta.url)
+
+  let sendResponse = content => {
```

Comment:
Let’s move function out of `handleMain` since it static

Reply:
I moved it to `lib/send-sesponse.js`